paper-chip-input.html

```diff
@@ -207,6 +206,12 @@
                 }
             }
 
+            static get observers() {
+                return [
+                    "_onValuesChanged(values, values.length)",
```

Comment:
To observe all changes to a given path and its subproperties we should use:
`_onValuesChanged(values, values.*)`

[docs](https://www.polymer-project.org/2.0/docs/devguide/model-data#specifying-paths)